src/PKSim.Assets/PKSim.Assets.csproj

```diff
@@ -10,6 +10,7 @@
     <OutputPath>bin\$(Configuration)</OutputPath>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
     <NoWarn>1591</NoWarn>
+    <Version Condition="'$(Version)' == ''">12.1.0.0</Version>
```

Comment:
12.1.0.0 or 12.1.0 ?

Reply:
I think you define all 3 digits. That's what we had for local builds of 11.3 -> 11.3.0.0
![image](https://github.com/user-attachments/assets/25acddde-4fc0-4814-825b-a3fef8cfa31c)

Reply:
![image](https://github.com/user-attachments/assets/4e0f9669-bd8d-48b4-be01-0a44a5553337)
That's where it makes a difference.

Reply:
Boom